requirements/requirements.txt

```diff
@@ -4,4 +4,5 @@ apache-airflow-providers-airbyte==3.2.0
 apache-airflow-providers-slack==7.2.0
 virtualenv==20.17.1
 
-dbt-core
\ No newline at end of file
+dbt-core
+pandas
```

Comment:
might recommend adding the version as well since it's already mentioned in the constraints file. 
mentioning the version makes it faster. I've kept it lose for DBT core so that pip can figure out a compatible version based on other dependencies and constraints. 

https://raw.githubusercontent.com/apache/airflow/constraints-2.6.3/constraints-3.10.txt

Reply:
👍